src/util/getCommandState.ts

```diff
@@ -16,6 +16,27 @@ const tags = {
   italic: createTag('i'),
   underline: createTag('u'),
   strikethrough: createTag('strike'),
+  foreColor: createTag('font'),
+  backColor: createTag('span'),
```

Comment:
Why does foreColor use `font` and backColor use  `span`?

Reply:
Because document.execCommand('foreColor') adds <font> tag and document.execCommand('backColor') adds <span> tag.

Reply:
Can we either use a single span tag with both styles, or two span tags? It seems pretty hacky to mix `font` and `span`.

Reply:
Ok, I will check.